source/features/resolve-conflict.tsx

```diff
@@ -0,0 +1,186 @@
+import React from 'dom-chef';
+import CodeMirror from 'codemirror';
```

Comment:
Here you’re importing CodeMirror into the file. We only need its types

Reply:
Wow, surprisingly this was not actually importing CodeMirror. I just looked at the bundle it generated.

@sindresorhus is there a way to specify "import JUST the types" to avoid any confusion and any involuntary importing of actual code? An `import` like this is one line away from importing the whole library. Example

```diff
+ console.log(CodeMirror);
```

```
           background.js ⏤  4.2 kB (+2 B)
              content.js ⏤  124 kB (+79.2 kB)
              options.js ⏤  2.82 kB
             options.css ⏤  658 B
            options.html ⏤  956 B
 browser-polyfill.min.js ⏤  3.1 kB
             content.css ⏤  7.89 kB
```

Reply:
Oh now I get your point 😅, I thought you mean we need to import just what we need.  
I had same mistake with `CodeMirror.Pos` and it increased bundle size, then I found the issue and create `pos` function
```typescript
pos(line: number, index: number) {
	// @ts-ignore
	const {CodeMirror} = window;
	return new CodeMirror.Pos(line, index);
}
```

Reply:
Yeah the code was actually fine, but `import ... from 'codemirror'` totally _looks like_ it imports the code, and potentially it can. If possible I'd like to avoid that

Reply:
Fixed! https://github.com/sindresorhus/refined-github/pull/1882/commits/2743dff6da1a5e3054b291667a51c89887285704